src/universe/qual/Bottom.java

```diff
@@ -23,9 +21,9 @@
  */
 @Documented
 @Retention(RetentionPolicy.RUNTIME)
-@Target({ ElementType.TYPE_PARAMETER, ElementType.TYPE_USE })
-@TargetLocations({TypeUseLocation.EXPLICIT_LOWER_BOUND,
-        TypeUseLocation.EXPLICIT_UPPER_BOUND})
+@Target({ElementType.TYPE_PARAMETER, ElementType.TYPE_USE})
```

Comment:
I don't understand this change... Should the extra space be used everywhere? `@Target({})` does not have the extra space, while `@TargetLocations({  })` has extra spaces.

Reply:
I noticed that there is spacing in most of the other attributes, with the exception of `@Target`
I've since applied it to the other qualifiers as well in ee0e7ab26a273c1f72c140262cac54775a2ee90c

I'm not sure about the coding style conventions, so I've enforced these arbitrarily -- is that alright?

Reply:
Thanks for the changes! It doesn't matter whether the spaces should be added. I think it is ok to have spaces for now. We may add a `google-java-format` plugin in the future to do a whole-program-reformat to enforce the formatting rule.

Reply:
In the other type systems, it seems that there are no extra spaces --- e.g., https://github.com/typetools/checker-framework/blob/master/checker/src/main/java/org/checkerframework/checker/initialization/qual/FBCBottom.java#L24

But I think reformatting is not an important task for now, after we finish the code updating and refactoring, we can think about the code format. So I think it is ok to keep your changes. In my last review, I was just wondering whether we should unify to use extra spaces --- it is confusing that somewhere we have extra spaces, while somewhere we don't.